sdk/appconfiguration/app-configuration/samples/v1/javascript/package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "@azure/app-configuration": "latest",
     "dotenv": "latest",
-    "@azure/identity": "^4.2.1",
+    "@azure/identity": "^4.0.1",
```

Comment:
isn't 4.5.0 the latest? why downgrading?

Reply:
I believe the `samples publish` script is using the same version we have in the main `package.json`. Updated the main one to `4.4.1`